game/definitions.rpy

```diff
@@ -1603,19 +1603,6 @@ init -100 python in jn_utils:
         """
         return list(store.player)[0]
 
-    def getPlayerFinal(repeat_times=0):
-        """
-        Returns the last letter of the player's name.
-
-        OUT:
-            Last letter of the player's name
-        """
-        player_final = list(store.player)[len(store.player) - 1]
-        for i in range(repeat_times):
-            player_final += list(store.player)[len(store.player) - 1]
-
-        return player_final
-
```

Comment:
This shouldn't have been removed! Looks like something that was caught up when the merges to bring this branch up to date got messy. The glasses event relies on this, so we would need it restoring!